frontend/src/main/webapp/pagevisit.js

```diff
@@ -0,0 +1,26 @@
+function PageData() {
```

Comment:
was there a reason why you capitalized this function?

Reply:
No, I just forget to change it